cms/envs/test_docker.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is needed for me to test locally with Docker devstack. I will move it to its own commit.